docs/root/configuration/listeners/network_filters/local_rate_limit_filter.rst

```diff
@@ -25,7 +25,7 @@ processed by the filter utilizes a single token, and if no tokens are available,
 be immediately closed without further filter iteration.
 
 .. note::
-  In the current implementation each filter and filter chain has an independent rate limit.
+  In the current implementation each filter chain has an independent rate limit.
```

Comment:
Isn't this comment redundant? Since the filter is configured in a filter chain, it's supposed to be independent of other rate limit filters configured in other filter chains.

Reply:
Yeah I think this should be each filter vs. filter chain, as a filter chain could have multiple filters with independent limits.

/wait

Reply:
Ah yes, I mixed up filter (instance) with filter configuration.